.gitignore

```diff
@@ -14,5 +14,4 @@ package-lock.json
 
 .DS_STORE
 coverage
-dist
```

Comment:
Temporary change so that I can test https://github.com/MetaMask/metamask-extension/pull/13865.